pkg/registry/apis/query/query.go

```diff
@@ -160,6 +160,7 @@ func (r *queryREST) Connect(connectCtx context.Context, name string, _ runtime.O
 		// Actually run the query
 		rsp, err := b.execute(ctx, req)
 		if err != nil {
+			b.log.Error("hit unexpected error while executing query, this will show as an unhandled k8s status error", "err", err)
```

Comment:
this is neat! i recommend just making a PR with this single change, we can quickly approve and merge it.

Reply:
ok done here: https://github.com/grafana/grafana/pull/100147